### ISSUE TITLE
Bootstrap fails trying to download vswhere

### ISSUE THREAD

**Issue by coryan** [Microsoft/vcpkg#2864]
I get the following error when running the boostrap script on a AppVeyor build:

```console
> .\bootstrap-vcpkg.bat
Downloading vswhere...
Exception calling "DownloadFile" with "2" argument(s): "The request was aborted: Could not create SSL/TLS secure channel."
```

Note that this only happens if the build has **not** cached vswhere from a previous run.  It seems github (or S3 where the github release files seem to be hosted), requires TLS 1.2.  The following patch fixes the problem for me, but it is likely to be the Wrong Way :tm: to fix the problem:

```diff
index ec3cdc83..29baa07e 100644
--- a/scripts/fetchDependency.ps1
+++ b/scripts/fetchDependency.ps1
@@ -40,6 +40,7 @@ function SelectProgram([Parameter(Mandatory=$true)][string]$Dependency)
     }
     elseif($Dependency -eq "vswhere")
     {
+        [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
         $requiredVersion = "2.3.2"
         $downloadVersion = "2.3.2"
         $url = "https://github.com/Microsoft/vswhere/releases/download/2.3.2/vswhere.exe"
```

**Comment by LeonineKing1199** [comment]
Can confirm same issue as well.

**Comment by longnt0707** [comment]
I have the same issue with https://github.com/Microsoft/vcpkg/commit/4a25962307ae3317c94ce1e923ed8aea713931a3

**Comment by alexkaratarakis** [comment]
We just released a hotfix in latest master (ce53ea00118154f240e7b5cb1856fd85b45dd216, PR #2863).

**Comment by coryan** [comment]
Thanks for the quick turnaround, it is working for me.
